src/components/Breadcrumb/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Fragment } from 'react'
+import { PAGE_TYPE } from 'constants'
+import { useGlobalContext } from 'providers/GlobalProvider'
+import Box from '@material-ui/core/Box'
+import {
+  AiOutlineBranches,
+  AiOutlineFile,
+  AiOutlineAppstore,
+  AiOutlineGithub,
+  AiOutlinePullRequest,
+  AiOutlineFieldNumber,
+  AiOutlineRight,
+} from 'react-icons/ai'
+
+const Breadcrumb = ({
+  pageType,
+  owner,
+  repo,
+  commit,
+  pull,
+  branch,
+  filePath,
+}) => {
+  const { togglePullMenu, togglePullCommitMenu } = useGlobalContext()
+
+  const items = [
+    { text: owner, icon: AiOutlineGithub },
+    { text: repo, icon: AiOutlineAppstore },
```

Comment:
How about linking to the default branch code page?

Reply:
updatedd